src/Command/MigrateToJsonTypeCommand.php

```diff
@@ -33,18 +48,21 @@ public function configure()
 
     public function execute(InputInterface $input, OutputInterface $output)
     {
+        if (null === $this->entityManager) {
+            throw new \LogicException('The ORM feature is disabled.');
```

Comment:
```suggestion
            throw new \LogicException(
                'This command could not be executed since one of its dependencies is missing.'
                .' Is the service "doctrine.orm.entity_manager" available?'
            );
```